mobile/README.md

```diff
@@ -16,11 +28,22 @@
 
 ### Go mobile
 
-Install [gomobile](https://github.com/golang/go/wiki/Mobile) and initialize it:
+- Install [gomobile](https://github.com/golang/go/wiki/Mobile):
 
 ```shell
 ⛰  go install golang.org/x/mobile/cmd/gomobile@latest
-⛰  gomobile init
+```
+
+- Install `gobind`
+
+```shell
+⛰  go install golang.org/x/mobile/cmd/gobind@latest
+```
+
+- Install `falafel`
```

Comment:
I don't think `falafel` needs to be installed anymore since this is now done inside the docker container when running `make mobile`.

Reply:
if you're sure about that then I'll remove it, no problem.